keyboards/studiokestra/nue/keymaps/default/keymap.c

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2021 Studio Kestra
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15